### ISSUE TITLE
Job does not get latest executing date

### ISSUE THREAD

**Issue by svenkubiak** [svenkubiak/mangooio#339]
When a scheduler job is triggered via the admin pages it does not get set the latest execution date. Also the jobs gets executed.

**Comment by svenkubiak** [comment]
Won't fix as this seems to deeply implemented into quartz functionality and how we currently execute a single job only once.
